Tools/WebKitTestRunner/Configurations/WebKitTestRunnerApp.xcconfig

```diff
@@ -37,13 +37,19 @@ STRIP_STYLE = debugging;
 
 SKIP_INSTALL[sdk=macosx*] = YES;
 
-EXCLUDED_SOURCE_FILE_NAMES[sdk=macosx*] = ios/* AppDelegate.m;
-EXCLUDED_SOURCE_FILE_NAMES[sdk=appletv*] = ios/Launch.storyboard;
-EXCLUDED_SOURCE_FILE_NAMES[sdk=watch*] = ios/Launch.storyboard;
+EXCLUDED_SOURCE_FILE_NAMES[config=Production] = *.appex
+EXCLUDED_SOURCE_FILE_NAMES[sdk=macosx*] = ios/* AppDelegate.m *.appex;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=appletv*] = ios/Launch.storyboard *.appex;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=watch*] = ios/Launch.storyboard *.appex;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=xr*] = *.appex;
```

Comment:
As written, I think the `config` and `sdk` variations will override each other. You could add an `$(inherited)` to both lines so that whichever one is evaluated first does not override the other.